der/src/decode.rs

```diff
@@ -34,11 +34,24 @@ pub trait Decode<'a>: Sized + 'a {
     }
 
     /// Parse `Self` from the provided DER-encoded byte slice.
+    ///
+    /// Returns [`ErrorKind::TrailingData`] if message is incomplete.
```

Comment:
`ErrorKind` isn't in scope. Maybe this?

```suggestion
    /// Returns [`crate::ErrorKind::TrailingData`] if message is incomplete.
```